src/lua/api-gateway/redis/redisHealthCheck.lua

```diff
@@ -27,24 +27,24 @@
 -- To change this template use File | Settings | File Templates.
 --
 
-
--- Base class for redis health check to get the healthy node
-
-local base = require "api-gateway.validation.base"
-
-local HealthCheck = {}
+local RedisHealthCheck = {}
 local DEFAULT_SHARED_DICT = "cachedkeys"
+local HEALTHY_REDIS_UPSTREAM_KEY_PREFIX = "healthy_redis_upstream:"
 
-function HealthCheck:new(o)
+function RedisHealthCheck:new(o)
```

Comment:
is there any possibility that we can an add an UT for this module?

Reply:
Yes, will do now.